tests/components/blebox/test_config_flow.py

```diff
@@ -0,0 +1,180 @@
+"""Test Home Assistant config flow for BleBox devices."""
+
+import blebox_uniapi
+import pytest
+
+from homeassistant import config_entries, data_entry_flow
+from homeassistant.components.blebox import config_flow
+from homeassistant.setup import async_setup_component
+
+from .conftest import mock_config, mock_only_feature, setup_product_mock
+
+from tests.async_mock import DEFAULT, AsyncMock, PropertyMock, patch
+
+
+async def empty_config_flow_from_user(hass):
```

Comment:
These helpers make it harder to see what the tests are doing. Please remove them.

Reply:
Done. (Though I personally would've at least refactored out the common `data=` parameter.)